### ISSUE TITLE
Hi，do you get the final result?

### ISSUE THREAD

**Issue by LiuzhuForFun** [aDecisionTree/underwater_detection_docker#1]
Only one team in the tab page of track 1.3. http://cvpr2020.ug2challenge.org/leaderboard20_t1.html, Is that your the winner?

**Comment by aDecisionTree** [comment]
No, it is not our's team. I guess maybe the testing is going on and the leaderboard is incomplete.

**Comment by LiuzhuForFun** [comment]
Thanks a lot!

**Comment by aDecisionTree** [comment]
With no news for so many days, I guess only one team passed the test. Our docker may have failed.

**Comment by LiuzhuForFun** [comment]
> With no news for so many days, I guess only one team passed the test. Our docker may have failed.

That's a terrible news!

**Comment by LiuzhuForFun** [comment]
Our team also put a lots of efforts to build the method. Maybe a neccessary email to the offical organizers is needed. What's your viewpoint of this aspect?

**Comment by aDecisionTree** [comment]
We don't want to send an email.

**Comment by aDecisionTree** [comment]
We just participate for fun. :-)
